scripts/bot-deployment/PushBotHotFix.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ]; then
```

Comment:
note that this script is currently quite fragile as you need to specify a bot name that currently exists. This should be updated in the future to ensure that the bot name specified is one of the currently deployed bots.

I'm not going to include this in this PR because it will be a fair amount of bash logic and will also need to change the other update script. This is left as an issue for future improvement in issue [1433](https://github.com/UMAprotocol/protocol/issues/1433)